packages/vuetify/src/components/VTabs/VTabs.sass

```diff
@@ -127,7 +127,7 @@
     .v-tabs-bar__content > *:first-child:not(.v-tabs-slider-wrapper)
       margin-left: auto
 
-    .v-tabs-slider-wrapper + *
+    .v-tabs-bar__content > .v-tab:first-of-type
```

Comment:
`:first-of-type` does not work on classes afaik

Reply:
the solution works for me, bit i can live with my hotfix